### PR TITLE
fix children query, it was not filtered

### PR DESCRIPTION
**Pull request by avalla**
the children were not filtered

### CODE REVIEW

**Comment by Aurelsicoko** [comment]
Thanks for your PR ! I don't remember why there is a `where` condition for children query. I will test your PR, before the merge.

File: lib/adapter.js [hunk 1]
```diff
@@ -113,9 +113,7 @@ module.exports = {
                 const criteria = {};
                 criteria[rules.via.toLowerCase()] = object[collection.primaryKey];
 
-                return self.defaults.queryFields[rules.collection.toLowerCase() + 's'].resolve(object, {}, {
-                  where: criteria
-                });
+		return self.defaults.queryFields[rules.collection.toLowerCase() + 's'].resolve(object, criteria);
```
Comment (Aurelsicoko) [inline lib/adapter.js]: Can you fix the indentation on this line, please?

Comment (avalla) [reply lib/adapter.js]: fixed, sorry, I pasted the new line on vi and didn't care about indentation

**Comment by Aurelsicoko** [comment]
@avalla Can you show me a code snippet where the children queries are not filtered?

**Comment by avalla** [comment]
I tried on a simple query on graphiql, without the fix the children were not filtered. Now I removed everything, I'm building the schema alone.
However I had a similar result:

Data:

```
{
  [
    parent : 1,
    children: [
      1,
      2
    ]
  ],
  [
    parent : 2,
    children: [
      3,
      4
    ]
  ],
}
```

Query:

```
{
  parents {
    parent
    children
  }
}
```

Result:

```
{
  [
    parent : 1,
    children: [
      1,
      2,
      3,
      4
    ]
  ],
  [
    parent : 2,
    children: [
      1,
      2,
      3,
      4
    ]
  ],
}
```

**Comment by avalla** [comment]
I think that there is also a bug on the type field. I received it as null, with an error because it is a non null type, I fixed the issue adding this on line 133, but I don't think that is the right solution:

```
resolve: function() { return _.capitalize(collectionIdentity) }
```

**Comment by Aurelsicoko** [comment]
Thanks for your quick answer! I don't have enough time to well-tested, I will merge your PR to unblock you on your project.

Your second fix looks great! Feel free to submit another PR :)

**Pull request merged**